modules/uninstall-cluster-logging-operator.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-uninstall.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="uninstall-cluster-logging-operator_{context}"]
+= Uninstalling the {logging}
+
+You can stop log aggregation by deleting the {clo} and the `ClusterLogging` custom resource (CR).
```

Comment:
```suggestion
You can stop aggregating logs by deleting the {clo} and the `ClusterLogging` custom resource (CR).
```
- Maybe better to verb-ify this phrase? Just a suggestion.
- Is "aggregate" the right word? Or should it be "collect"? In my mind, they mean two slightly different things.

Reply:
I'd say aggregate is right; collection is specifically done by the log collector I think.